### ISSUE TITLE
flet publish broken

### ISSUE THREAD

**Issue by wschume** [flet-dev/flet#1593]
**Description**

Using `flet publish` does not create a functioning static web site. The web server runs successfully, however, now contents of the page is displayed.

With the minimum working example:

```python
import flet as ft


def main(page: ft.Page):
    page.add(ft.Text("Simple App"))

    page.update()


if __name__ == "__main__":
    ft.app(target=main)
```

the application can be run using the desktop as well as the web browser version. However, after running

`flet publish simple_app.py`

the web site located in the `dist` directory. Running `python -m http.server --directory dist` launches the web server. But loading the site in the browser hangs indefinitely. The log produces is shown below:

```
Serving HTTP on 0.0.0.0 port 8000 (http://0.0.0.0:8000/) ...
127.0.0.1 - - [14/Jul/2023 22:13:47] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [14/Jul/2023 22:13:47] "GET /flutter_service_worker.js?v=1209606426 HTTP/1.1" 200 -
127.0.0.1 - - [14/Jul/2023 22:13:47] "GET /main.dart.js HTTP/1.1" 200 -
127.0.0.1 - - [14/Jul/2023 22:13:47] "GET /assets/AssetManifest.json HTTP/1.1" 200 -
127.0.0.1 - - [14/Jul/2023 22:13:47] "GET /index.html HTTP/1.1" 200 -
127.0.0.1 - - [14/Jul/2023 22:13:47] "GET /assets/FontManifest.json HTTP/1.1" 200 -
127.0.0.1 - - [14/Jul/2023 22:13:48] "GET /flutter_service_worker.js?v=1209606426 HTTP/1.1" 304 -
127.0.0.1 - - [14/Jul/2023 22:13:48] "GET /flutter_service_worker.js?v=1209606426 HTTP/1.1" 304 -
127.0.0.1 - - [14/Jul/2023 22:13:50] "GET /app.tar.gz HTTP/1.1" 200 -
```

**Describe the results you expected:**

I expect to see the web site, when loading it in the browser.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.8.2
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: 
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: /data/niklas/PycharmProjects/pythonProject/venv/lib/python3.10/site-packages
Requires: copier, flet-runtime, packaging, pydantic, qrcode, watchdog, websocket-client, websockets
Required-by: 
```

**Operating system**:

**Additional environment details:**

**Comment by FeodorFitsner** [comment]
Try removing `if __name__ == "__main__":` condition.

**Comment by ndonkoHenri** [comment]
Just from facing thesame issue and Feodor's suggestion fixed it.
Thanks Feodor.

**Comment by wschume** [comment]
I can confirm, that this fixes the issue. I now only have the remaining problem that my actual application does not appear to work with `flet publish`. But this would then be a separate ticket.
Thank you for the quick help!

**Comment by wschume** [comment]
I can confirm, that this fixes the issue. I now only have the remaining problem that my actual application does not appear to work with flet publish. But this would then be a separate ticket.
Thank you for the quick help!
